doc/api/util.md

```diff
@@ -89,16 +89,16 @@ For example:
 
 ```js
 const util = require('util');
-const debuglog = util.debuglog('foo');
+const debuglog = util.debuglog('foo-bar');
 
 debuglog('hello from foo [%d]', 123);
 ```
 
-If this program is run with `NODE_DEBUG=foo` in the environment, then
+If this program is run with `NODE_DEBUG=foo*` or `NODE_DEBUG=foo-bar` in the environment, then
 it will output something like:
 
 ```txt
-FOO 3245: hello from foo [123]
+FOO-BAR 3245: hello from foo [123]
```

Comment:
It would be nice to have another example specifically for the wildcard. Including counter examples. Especially people who are new to coding might want to have more explicit explanations.

Reply:
thx, added.